src/main/com/radek/example1/WholeNumberGeneratorByRadekApplication.java

```diff
@@ -4,13 +4,14 @@
 import java.util.Random;
 import java.util.Scanner;
 
-public class Whole_RNG {
```

Comment:
Zadání bylo ať uděláš generátor a nebylo to nijak omezeno. Hodně sis to zkomplikoval, ale aspoň sis procvičil scanner. 
1 - Hodil jsem ti tam třídu, jak jsem si to zhruba představoval, tak zkus tu tvou taky zjednodušit. 
2 - V javě se používá camel syntax pro názvy  lokálních proměnných ... tzn. countArg, randomNumber
3 - klidně použij více metod, ale určitě ne jednu